doc/prop.xml

```diff
@@ -8,6 +8,83 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+    We will consider a digraph as a partial order with relation defined by
+    <A>x</A> <M>\leq</M> <A>y</A> if and only if <A>(x,y)</A> is an edge.
```

Comment:
We use `<A>` tags only for the specific arguments defined for the function that is being documented.

Since `x` and `y` are not arguments to `IsPartialOrderDigraph`, so they shouldn't be in these tags. Similarly for `(x,y)`.